indico/modules/events/api.py

```diff
@@ -289,6 +290,15 @@ def _iterate_objs(objIds):
     def serialize_events(self, events):
         return map(self._build_event_api_data, events)
 
+    def _serialize_access_list(self, obj):
+        data = {'users': [], 'groups': []}
+        for manager in obj.get_access_list():
+            if hasattr(manager, 'user'):
```

Comment:
why, what's wrong with that? I have one less import this way and there is no extra queries or anything.